.buildkite/pipeline.yml

```diff
@@ -0,0 +1,47 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  plugins: &common_plugins
+  - &bash_cache automattic/bash-cache#v1.3.2: ~
+  # Common environment values to use with the `env` key.
+  env: &common_env
+    IMAGE_ID: xcode-12.5.1
+
+# This is the default pipeline – it will build and test the app
+steps:
+
+  #################
+  # Validate Podspec
+  #################
+  - label: "🔬 Validate Podspec"
+    key: "validate"
+    command: "validate_podspec"
+    env: *common_env
+    plugins: *common_plugins
+
+  #################
+  # Lint
+  #################
+  - label: "🧹 Lint"
+    key: "lint"
+    command: .buildkite/lint.sh
+    env: *common_env
```

Comment:
Given this step seems to run into Docker, and thus probably doesn't have a hard requirement for the host to be a Mac nor to use an Xcode VM (I _think_ that the `IMAGE_ID` would be unused in this case give this uses Docker and an `agents: { queue: "default" }` not a Mac host, right?), why do we need this `env:` to be `{ IMAGE_ID: "xcode-12.5.1"`?

Sure it probably doesn't have any impact to set it if it's ultimately unused, but for clarity and avoiding confusion I think we'd better not set it if it's not needed.

Reply:
Yep agreed – removed in 8eec5dbdf6abe5ef681b7488ddbc57ad6ca53835